website/content/docs/connect/ca/vault.mdx

```diff
@@ -80,11 +69,12 @@ connect {
 
 </CodeTabs>
 
-The configuration options are listed below.
+## Configuration Reference
```

Comment:
At some point, we should just make a pass over all the pages to change the capitalization.

For any _new_ headers I create, I'm using the style guide. But for content I'm not editing (or only shuffling on the page), I haven't been touching the existing header to make them compliant (because I'm focused on a particular change).

This probably creates a lot of back and forth for both reviewers and authors. If we update all the content at once, it should be easier for everyone going forward.